target/create.ps1

```diff
@@ -67,13 +71,15 @@ try {
         }
         # Example of replacing the placeholder with actual code:
         # Retrieve user details using an API call and store the result in $correlatedAccount
-        # $correlatedAccount = Invoke-RestMethod @splatGetUserParams
+        # $correlatedAccount = (Invoke-RestMethod @splatGetUserParams) | Select-Object -First 1
```

Comment:
Door hier `| Select-Object -First 1` te doen, krijg je hierna geen `Multiple Found` meer.

Je kunt pas een Select-Object zodra je weet dat er maar 1 object terug is gekomen. Het zou dus in de conditie `} elseif ($correlatedAccount.Count -eq 1) {` kunnen.

Al vraag ik me af of dit in het template terug moet komen.